package.json

```diff
@@ -18,6 +18,7 @@
   "author": "Grafana Labs",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@grafana/experimental": "^0.0.2-canary.38",
```

Comment:
what happens if Athena is run with a lower version of Grafana? (that does not include the latest experimental package)

Reply:
yeah good shout, better make this a dependency. I wonder what would happen though if grafana is running a version of the experimental package that is lower than the plugin specifies..?

Reply:
I belive it gets ignored, no matter if it's a dependency or a devdep (there is no reference to the deps in the `dist` folder, when you build a plugin).

Reply:
Should be fine since experimental package is not specified as an external dep in core grafana

Reply:
oh, I didn't know it works that way, interesting!